### PR TITLE
Integrated material-ui@next and added a sample code.

### LINKED ISSUE #14: Setting up SASS, or Material UI/ Bootstrap in the frontend code
Hi All, Meanwhile wireframes are getting ready , we can integrate SASS, or Material UI/ Bootstrap in  our code. Provide feedback. I can start on this once stack is finalised.

### PR DESCRIPTION
**Pull request by nikitasharma2692**
Fixes #14 

#### Checklist

- [x] I have read the Contribution & Best practices Guide and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added/updated necessary tests/documentation (if applicable)

#### Changes proposed in this pull request:

-integrated material-ui@next and added its sample code in the App.js.
-

### CODE REVIEW

**Comment by ritikrishu** [comment]
Please consider filling up the PR template.
@nikitasharma2692 I see you are self-motivated and one of the most active contributors to the project. Thank you for your contribution! ❤️

File: client/src/withRoot.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { MuiThemeProvider, createMuiTheme } from 'material-ui/styles';
+import purple from 'material-ui/colors/purple';
+import green from 'material-ui/colors/green';
+import CssBaseline from 'material-ui/CssBaseline';
+
+// A theme with custom primary and secondary color.
+// It's optional.
+const theme = createMuiTheme({
+    palette: {
```
Comment (ritikrishu) [inline client/src/withRoot.js]: Just curious how did you choose the initial colors? Are these from the branding samples on #5 ?

**Comment by nikitasharma2692** [comment]
Above colours are just to give hint on how things will work. I have already created a issue #23 which will replace the code.

**Comment by nikitasharma2692** [comment]
Will fill in the PR template

**Comment by palnabarun** [comment]
@nikitasharma2692 Reminder: Please fill in the PR template. Everything else look good.

**Comment by ritikrishu** [comment]
I think the issue number at the start of PR title was a good move! Username after that however, was not necessary 👍

**Comment by ritikrishu** [comment]
This PR is to address #14  right?
Does it contains SASS setup as well?

**Comment by nikitasharma2692** [comment]
Yes it contains SASS as well

**Comment by realslimshanky** [comment]
Thanks @nikitasharma2692 for your contribution :taco:

File: client/package.json [hunk 2]
```diff
@@ -12,5 +13,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "node-sass": "^4.8.3"
   }
 }
```
Comment (realslimshanky) [inline client/package.json]: We need a newline character here on line 16.

File: client/src/App.js [hunk 3]
```diff
@@ -1,21 +1,68 @@
...
+                <Button variant="raised" color="secondary" onClick={this.handleClick}>
+                    Test Popup
+                </Button>
+            </div>
+        );
+    }
 }
 
-export default App;
+export default withRoot(withStyles(styles)(App));
```
Comment (realslimshanky) [inline client/src/App.js]: We need one newline character here at the end of line 68

File: client/src/withRoot.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                <CssBaseline />
+                <Component {...props} />
+            </MuiThemeProvider>
+        );
+    }
+
+    return WithRoot;
+}
+
+export default withRoot;
```
Comment (realslimshanky) [inline client/src/withRoot.js]: We need a newline character here at the end of the line 40.

**Pull request merged**